README.md

```diff
@@ -164,6 +164,13 @@ Here is the list of the currently available helpers:
   option (defaults to `en-EN`).
   eg. `100000` will be formatted as `100 000` with `en-EN`
 
+Here is the syntax of a helper (`render` is using `search.templatesConfig.compileOptions`):
+```js
+search.templatesConfig.helpers.makeTitle = function (text, render) {
+  return '<h1>' + render(text) + '</h1>';
```

Comment:
We need a note about being able to access "this.prop" this referring to the `data`. This is a nice addition.